clients/ui/frontend/package.json

```diff
@@ -103,12 +103,12 @@
   "dependencies": {
     "@emotion/react": "^11.14.0",
     "@emotion/styled": "^11.14.0",
-    "@patternfly/patternfly": "^6.1.0",
-    "@patternfly/react-core": "6.1.0",
-    "@patternfly/react-icons": "6.1.0",
-    "@patternfly/react-styles": "6.0.0",
-    "@patternfly/react-table": "6.1.0",
-    "@patternfly/react-templates": "6.1.0",
+    "@patternfly/patternfly": "^6.2.0",
+    "@patternfly/react-core": "^6.2.0",
+    "@patternfly/react-icons": "^6.2.0",
+    "@patternfly/react-styles": "^6.2.0",
+    "@patternfly/react-table": "^6.2.0",
+    "@patternfly/react-templates": "^6.2.0",
```

Comment:
this mods seems to be coming from other commits and unrelated to $subject , do you need to rebase your PR with only necessary commits maybe?